common/persistence/cassandra/cassandraVisibilityPersistence.go

```diff
@@ -25,165 +25,26 @@ import (
-
-	templateCreateWorkflowExecutionClosed = `INSERT INTO closed_executions ` +
-		closedExecutionColumnsForInsert +
-		`VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)`
-
-	templateCreateWorkflowExecutionClosedWithTTLV2 = `INSERT INTO closed_executions_v2 ` +
-		closedExecutionColumnsForInsert +
-		`VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?) using TTL ?`
-
-	templateCreateWorkflowExecutionClosedV2 = `INSERT INTO closed_executions_v2 ` +
-		closedExecutionColumnsForInsert +
-		`VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)`
-
-	templateGetClosedWorkflowExecutions = `SELECT ` + closedExecutionColumnsForSelect +
-		`FROM closed_executions ` +
-		`WHERE domain_id = ? ` +
-		`AND domain_partition IN (?) ` +
-		`AND start_time >= ? ` +
-		`AND start_time <= ? `
-
-	templateGetClosedWorkflowExecutionsByType = `SELECT ` + closedExecutionColumnsForSelect +
-		`FROM closed_executions ` +
-		`WHERE domain_id = ? ` +
-		`AND domain_partition = ? ` +
-		`AND start_time >= ? ` +
-		`AND start_time <= ? ` +
-		`AND workflow_type_name = ? `
-
-	templateGetClosedWorkflowExecutionsByID = `SELECT ` + closedExecutionColumnsForSelect +
-		`FROM closed_executions ` +
-		`WHERE domain_id = ? ` +
-		`AND domain_partition = ? ` +
-		`AND start_time >= ? ` +
-		`AND start_time <= ? ` +
-		`AND workflow_id = ? `
-
-	templateGetClosedWorkflowExecutionsByStatus = `SELECT ` + closedExecutionColumnsForSelect +
-		`FROM closed_executions ` +
-		`WHERE domain_id = ? ` +
-		`AND domain_partition = ? ` +
-		`AND start_time >= ? ` +
-		`AND start_time <= ? ` +
-		`AND status = ? `
-
-	templateGetClosedWorkflowExecution = `SELECT ` + closedExecutionColumnsForSelect +
-		`FROM closed_executions ` +
-		`WHERE domain_id = ? ` +
-		`AND domain_partition = ? ` +
-		`AND workflow_id = ? ` +
-		`AND run_id = ? ALLOW FILTERING `
-
-	templateGetClosedWorkflowExecutionsV2 = `SELECT ` + closedExecutionColumnsForSelect +
-		`FROM closed_executions_v2 ` +
-		`WHERE domain_id = ? ` +
-		`AND domain_partition IN (?) ` +
-		`AND close_time >= ? ` +
-		`AND close_time <= ? `
-
-	templateGetClosedWorkflowExecutionsByTypeV2 = `SELECT ` + closedExecutionColumnsForSelect +
-		`FROM closed_executions_v2 ` +
-		`WHERE domain_id = ? ` +
-		`AND domain_partition = ? ` +
-		`AND close_time >= ? ` +
-		`AND close_time <= ? ` +
-		`AND workflow_type_name = ? `
-
-	templateGetClosedWorkflowExecutionsByIDV2 = `SELECT ` + closedExecutionColumnsForSelect +
-		`FROM closed_executions_v2 ` +
-		`WHERE domain_id = ? ` +
-		`AND domain_partition = ? ` +
-		`AND close_time >= ? ` +
-		`AND close_time <= ? ` +
-		`AND workflow_id = ? `
-
-	templateGetClosedWorkflowExecutionsByStatusV2 = `SELECT ` + closedExecutionColumnsForSelect +
-		`FROM closed_executions_v2 ` +
-		`WHERE domain_id = ? ` +
-		`AND domain_partition = ? ` +
-		`AND close_time >= ? ` +
-		`AND close_time <= ? ` +
-		`AND status = ? `
-)
-
 type (
-	cassandraVisibilityPersistence struct {
+	nosqlVisibilityManager struct {
```

Comment:
maybe also update the file name

Reply:
I guess that will be done when we move them to the nosql package? Currently they are hardcoded to use cassandra plugin.

Reply:
Yes that's the plan.